lib/spa-deploy/spa-deploy-construct.ts

```diff
@@ -73,6 +75,12 @@ export class SPADeploy extends cdk.Construct {
 
       if (this.globalConfig.ipFilter === true || isForCloudFront === true) {
         bucketConfig.publicReadAccess = false;
+        if (config.enableBlockPublicAccess === true) {
+          if (typeof config.blockPublicAccess === 'undefined') {
```

Comment:
why not just have the blockPublicAccess variable without the need for the enableBlockPublicAccess variable? Means you can just have the negated version of this if statement for when to set it